romanisim/ris_make_utils.py

```diff
@@ -123,7 +124,13 @@ def create_catalog(metadata=None, catalog_name=None, bandpasses=['F087'],
     if catalog_name is None:
         # Create a catalog from scratch
         # Create wcs object
-        twcs = wcs.get_wcs(metadata, usecrds=usecrds)
+        distortion_file = parameters.reference_file_names["distortion"]
+        if distortion_file is not None:
+            dist_model = roman_datamodels.datamodels.DistortionRefModel(distortion_file)
+            distortion = dist_model.coordinate_distortion_transform
+        else:
+            distortion = None
+        twcs = wcs.get_wcs(metadata, usecrds=usecrds, distortion=distortion)
```

Comment:
This is my mistake, but let's take this opportunity to document the distortion argument of get_wcs.

Reply:
I noticed this as well, but forgot to address it. I have added an entry.